src/test/java/com/systemmeltdown/robotlib/triggers/AxisThresholdTriggerTest.java

```diff
@@ -0,0 +1,74 @@
+package com.systemmeltdown.robotlib.triggers;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class AxisThresholdTriggerTest {
+  @Test
+	public void testGetReturnsTrueWhenHeld() {
+    XboxController controller = mock( XboxController.class );
```

Comment:
You should put common setup and tear down code in the setUp and tearDown methods instead of copying it into each test case. Those are run before and after each test method is run.